### PR TITLE
Audit and test opentelemetry-instrumentation-elasticsearch NoOpTracer…

### LINKED ISSUE #984: Audit and test opentelemetry-instrumentation-elasticsearch

### PR DESCRIPTION
**Pull request by Akochavi**
Add test for elasticsearch using NoOpTracerProvider

Fixes #984 

### How has this been tested?
tox -e test-instrumentation-elasticsearch

### CODE REVIEW

File: instrumentation/opentelemetry-instrumentation-elasticsearch/tests/test_elasticsearch.py [hunk 1]
```diff
@@ -413,3 +414,18 @@ def response_hook(span, response):
...
+        ElasticsearchInstrumentor().uninstrument()
+        ElasticsearchInstrumentor().instrument(tracer_provider=trace_api.NoOpTracerProvider())
+
+        request_mock.return_value = (
+            1,
+            {},
+            '{"found": false, "timed_out": true, "took": 7}',
+        )
+        es = Elasticsearch()
+        es.get(index="test-index", doc_type="_doc", id=1)
```
Comment (nemoshlag) [inline instrumentation/opentelemetry-instrumentation-elasticsearch/tests/test_elasticsearch.py:428]: I suggest validating es.get response value -> so we'll know for sure that no spans were created due to NoOpTracerProvider

File: instrumentation/opentelemetry-instrumentation-elasticsearch/tests/test_elasticsearch.py [hunk 2]
```diff
@@ -24,6 +24,7 @@
 
 import opentelemetry.instrumentation.elasticsearch
 from opentelemetry import trace
+from opentelemetry import trace as trace_api
```
Comment (shalevr) [inline instrumentation/opentelemetry-instrumentation-elasticsearch/tests/test_elasticsearch.py:27]: trace already imported in the upper line

Comment (Akochavi) [reply instrumentation/opentelemetry-instrumentation-elasticsearch/tests/test_elasticsearch.py]: Done

**Review by shalevr — APPROVED** [review]

File: instrumentation/opentelemetry-instrumentation-elasticsearch/tests/test_elasticsearch.py [hunk 3]
```diff
@@ -37,7 +37,7 @@
 
 major_version = elasticsearch.VERSION[0]
 
-if major_version == 7:
+if major_version  7:
```
Comment (srikanthccv) [inline instrumentation/opentelemetry-instrumentation-elasticsearch/tests/test_elasticsearch.py:40]: Is this a mistake?

Comment (Akochavi) [reply instrumentation/opentelemetry-instrumentation-elasticsearch/tests/test_elasticsearch.py:40]: I think so, I don't know who pushed this change but I will revert it

**Pull request merged**